pandas_vs_excel/pandas_vs_excel.ipynb

```diff
@@ -0,0 +1,640 @@
+       "        text-align: left;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>Student Name</th>\n",
+       "      <th>Exam 1 Score</th>\n",
+       "      <th>Exam 2 Score</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>Edwin Duncan</td>\n",
+       "      <td>83</td>\n",
+       "      <td>33</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>Steve Hayes</td>\n",
+       "      <td>69</td>\n",
+       "      <td>42</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>Cecilia Richards</td>\n",
+       "      <td>91</td>\n",
+       "      <td>73</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>Dorothy Harris</td>\n",
+       "      <td>53</td>\n",
+       "      <td>75</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>Jordan Davis</td>\n",
+       "      <td>76</td>\n",
+       "      <td>81</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "       Student Name  Exam 1 Score  Exam 2 Score\n",
+       "0      Edwin Duncan            83            33\n",
+       "1       Steve Hayes            69            42\n",
+       "2  Cecilia Richards            91            73\n",
+       "3    Dorothy Harris            53            75\n",
+       "4      Jordan Davis            76            81"
+      ]
+     },
+     "execution_count": 81,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# apply \"Vlookup\" in padas\n",
+    "exam_scores = exam_one.merge(exam_two, on='Student Name')\n",
+    "exam_scores.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "## Pivot Tables\n",
+    "\n",
+    "Pivot tables are another useful tool in Excel. It allows users to perform data aggregation; a new table is created that is a summary based on grouping of certain selected columns. Pivot tables can also be used to filter out rows from a table. In pandas, we can easily filter out rows from our `DatFrame` by using Boolean logic. For this example, we would like to determine the student's name that belong to section \"A\". This is done in pandas by first creating an array of True/False values. This array corresponds to which rows met the condition. We then use the resulting Boolean array to only call rows that meet our condition."
```

Comment:
"It" in "It allows" should be "They" (plural for pivot tables). The sentences are also a little confusing too. Maybe consider wording: "Pivot tables are used to aggregate and filter data. We can group data by certain values in a given column and we can filter out rows using boolean logic..." ??

Reply:
subject verb agreement -- I always mess that up. Conjugation in English (in my opinion)  is very subtle. I'll make this change and make the text clearer.